src/plugins/visualizations/server/migrations/visualization_saved_object_migrations.ts

```diff
@@ -1121,4 +1145,7 @@ export const visualizationSavedObjectTypeMigrations = {
     replaceIndexPatternReference,
     addDropLastBucketIntoTSVBModel
   ),
+  '7.15.0': (doc: SavedObjectUnsanitizedDoc) => doc,
```

Comment:
Same here

Reply:
As I know we cannot miss version here. We cannot just add `8.0.0`. But our logic should start work from `8.0.0`

Reply:
@alexwizp @VladLasitsa I tested this locally and couldn't get it to break even if these lines are omitted. Could you check how to replicate?

Reply:
Now I also cannot reproduce it, removed  line with unneeded versions.